### DISCUSSION TITLE
Error, when trying to run npm install.  npm ERR! While resolving: atlantis-ng@16.0.0

### DISCUSSION THREAD

**Discussion by yuriilevko99** [primefaces/community#613] [PrimeNG Templates] [answered]
When tried to build application on my second laptop - receive this error, while running npm install 

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: atlantis-ng@16.0.0
npm ERR! Found: @angular-devkit/build-angular@17.0.5
npm ERR! node_modules/@angular-devkit/build-angular
npm ERR!   dev @angular-devkit/build-angular@"^16.1.5" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! dev @angular-devkit/build-angular@"^16.1.5" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @angular/compiler-cli@16.2.12
npm ERR! node_modules/@angular/compiler-cli
npm ERR!   peer @angular/compiler-cli@"^16.0.0" from @angular-devkit/build-angular@16.2.10
npm ERR!   node_modules/@angular-devkit/build-angular
npm ERR!     dev @angular-devkit/build-angular@"^16.1.5" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Users\пк\AppData\Local\npm-cache\_logs\2023-12-05T09_03_07_753Z-eresolve-report.txt
npm ERR! A complete log of this run can be found in: C:\Users\пк\AppData\Local\npm-cache\_logs\2023-12-05T09_03_07_753Z-debug-0.log

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Try the steps below

1-) Delete package-lock.json file
2-) Delete node modules
3-) Clean npm cache by `npm cache clean --force` command
4-) Run `npm i`
